components/playlist/browser/resources/components/header.tsx

```diff
@@ -70,35 +132,16 @@ export default function Header ({ playlistId }: HeaderProps) {
             <ColoredIcon name='arrow-left' color={color.icon.default} />
           </StyledLink>
           <StyledPlaylistInfo
-            isDefaultPlaylist={playlist.id === 'default'}
+            isDefaultPlaylist={isDefaultPlaylist}
             itemCount={playlist.items.length}
             playlistName={playlist.name}
             totalDuration={0}
             nameColor={color.text.primary}
             detailColor={color.text.secondary}
           />
-          <ContextualMenuAnchorButton
-            items={[
-              { name: 'Rename', iconName: 'edit-box', onClick: () => {} },
-              {
-                name: 'Edit',
-                iconName: 'list-bullet-default',
-                onClick: () => {}
-              },
-              { name: 'Share', iconName: 'share-macos', onClick: () => {} },
-              {
-                name: 'Keep for offline playing',
-                iconName: 'cloud-download',
-                onClick: () => {}
-              },
-              {
-                name: 'Remove played contents',
-                iconName: 'list-checks',
-                onClick: () => {}
-              },
-              { name: 'Delete playlist', iconName: 'trash', onClick: () => {} }
-            ]}
-          />
+          {contextualMenuItems && (
+            <ContextualMenuAnchorButton items={contextualMenuItems} />
+          )}
```

Comment:
Acutally, contextMenuItems can't be null when playlist exists, but there are some mixed complaints from CI and dev env about this.

My dev thinks contextualMenuItems could be null but if I put non-null assertion `contextualMenuItems!`, then the mobile CI thinks the assertion isn't needed 😂

Reply:
This might be fixed if you define `contextualMenuItems` as:

```
const contextualMenuItems = playlist ? [
    ...
] : []
```

Reply:
Yeah, that'd be better! Thanks